common/predictive-text/unit_tests/in_browser/base.conf.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
     'cases/**/*.js', // Where the tests actually reside.
 
     // Include the generated worker code.
-    '../../embedded_worker.js',
+    '../../index.js',
```

Comment:
I assume this means we plan that for browser testing, we only plan to test the LMLayer as a whole, right?  Seems reasonable, since we do have the 'headless' mode for more specific unit tests - just want to make sure this is intentional.

Reply:
This is intentional! Headless unit tests will handle try to cover as much without invoking browser APIs like `Blob` and `URL.createObjectURL()`. Besides, in the browser, `index.js` exposes the main export from `embedded_worker.js` anyway.